cli/client.go

```diff
@@ -1153,14 +1153,21 @@ var clientRetrieveCmd = &cli.Command{
 						retrievalmarket.ClientEvents[evt.Event],
 						retrievalmarket.DealStatuses[evt.Status],
 					)
-				} else {
-					afmt.Println("Success")
-					return nil
 				}
 
 				if evt.Err != "" {
 					return xerrors.Errorf("retrieval failed: %s", evt.Err)
 				}
+
+				if !ok {
+					if evt.Status == retrievalmarket.DealStatusCompleted {
+						afmt.Println("Success")
+					} else {
+						afmt.Printf("saw final deal state %s instead of expected success state DealStatusCompleted", retrievalmarket.DealStatuses[evt.Status])
```

Comment:
```suggestion
						afmt.Printf("saw final deal state %s instead of expected success state DealStatusCompleted\n", retrievalmarket.DealStatuses[evt.Status])
```

Reply:
done.